app/models/appointment.rb

```diff
@@ -1,8 +1,5 @@
 class Appointment < ApplicationRecord
-  has_many :doctor
-  has_many :users
-
-  validates :description, presence: true
-  validates :time, presence: true
-  validates :date, presence: true
+  belongs_to :user
+  belongs_to :doctor
+  belongs_to :doctor_appointment_time
 end
```

Comment:
Please ensure that the models have proper validation to prevent the storage of inaccurate data in the database.